general/development/process/peer-review/accessibility-checklist.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Accessibility checklist
+tags:
+  - Processes
+  - Integration
+  - Accessibility
+sidebar_position: 4
+---
+
+<!-- cspell:ignore ACTIVITYNAME, Deque -->
+
+This document proposes a checklist for accessibility reviews. Not all aspects of accessibility can be checked automatically, so this checklist is intended to guide reviewers when doing an accessibility check to a patch.
+
+## Identify the Nature of Changes
+
+**Are the changes on the backend only (webservices, database updates, events, hooks...)?**
```

Comment:
```suggestion
- **Are the changes on the backend only (webservices, database updates, events, hooks...)?**<br/>
```

Reply:
To make it more readable, I think it would be better to display these questions in bullet lists (I've only changed the first one, but the other 2 questions should be changed too).